proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -3,9 +3,46 @@
 * Proposal: [SE-0293](0293-extend-property-wrappers-to-function-and-closure-parameters.md)
 * Authors: [Holly Borla](https://github.com/hborla), [Filip Sakel](https://github.com/filip-sakel)
 * Review Manager: [Chris Lattner](https://github.com/lattner)
-* Status: **Returned for revision**
+* Status: **Awaiting review**
```

Comment:
This has already been scheduled for review in 0a6a08cafc529c6d4c56c2bb4ceeee8a59d01828. Undo changes?

```suggestion
* Status: **Returned for revision**
```